docs/csharp/tutorials/intro-to-csharp/introduction-to-classes.md

```diff
@@ -90,17 +90,17 @@ var account = new BankAccount("<name>", 1000);
 Console.WriteLine($"Account {account.Number} was created for {account.Owner} with {account.Balance} initial balance.");
 ```
 
-Type `dotnet run` to see what happens.  
+Let's run what you've built so far. If you're using Visual Studio, Select *Start without debugging* from the *Run* menu. If you're using a command shell, type `dotnet run` in the directory where you've created your project.
```

Comment:
Style-guide, and command line instead of command shell. command line sounds more familiar.
```suggestion
Let's run what you've built so far. If you're using Visual Studio, Select **Start without debugging** from the **Run** menu. If you're using a command line, type `dotnet run` in the directory where you've created your project.
```